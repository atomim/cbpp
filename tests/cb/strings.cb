#define GENRE "dnb"
#define POST(genre) "post-"+genre

AddText "my favourite GENRE is " + GENRE
AddText "but even better is " + POST(GENRE)