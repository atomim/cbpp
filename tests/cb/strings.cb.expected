// #define GENRE "dnb"
// #define POST(genre) "post-"+genre

AddText "my favourite GENRE is " + "dnb"
AddText "but even better is " + "post-"+"dnb"